flow-api/src/main/java/com/sequenceiq/flow/api/FlowEndpoint.java

```diff
@@ -80,4 +86,21 @@ public interface FlowEndpoint {
     @ApiOperation(value = "Check if there is a running flow for flow id", produces = "application/json", notes = "Flow log operations",
             nickname = "hasFlowRunningByFlowId")
     FlowCheckResponse hasFlowRunningByFlowId(@PathParam("flowId") String flowId);
+
+    @GET
+    @Path("/check/flowIds")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Get flow logs by a list of flow ids - Input size max 50", produces = "application/json", notes = "Flow log operations",
+            nickname = "getFlowLogStatusesByFlowIds")
+    List<FlowLogResponse> getFlowLogStatusesByFlowIds(@NotNull @Size(max = 50, message = "Input size should not be over 50")
+        @QueryParam("flowIds") List<String> flowIds, Pageable pageable);
+
+    @GET
+    @Path("/check/chainIds")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Check if there is a running flow for chain ids - Input size max 50",
+            produces = "application/json", notes = "Flow check log operations",
+            nickname = "getFlowChainsStatusesByChainIds")
+    List<FlowCheckResponse> getFlowChainsStatusesByChainIds(@NotNull @Size(max = 50, message = "Input size should not be over 50")
+        @QueryParam("chainIds") List<String> chainIds, Pageable pageable);
```

Comment:
I checked our other pageable endpoints and we have page number and page size as a request parameter, and not `Pageable`. I haven't checked what's a swagger generated in case of pagable looks like, but the page number and size looks more strict to me, which I think could be better from API perspective, as you wouldn't have to validate and test every scenario

Reply:
Pageable is nothing but page number, size and sort together. sort can be set as null and page number is default to 0.

Reply:
this is how it will be passed in the url ?page=0&size=3&sort=id&direction=DESC

Reply:
they are not sent in the body - it comes through URL.

Reply:
as query param

Reply:
please test it, as it doesn't. I'm testing it and the object is null
also if you check the generated swagger, it's showing up as body:
![image](https://user-images.githubusercontent.com/16664623/209156698-5ce92ac6-4991-4fdd-8280-d4ddd8e4743a.png)